packages/frontend/src/components/common/PivotTable/index.tsx

```diff
@@ -60,6 +60,23 @@ const PivotTable: FC<PivotTableProps> = ({
     );
 
     const getRowTotalValueFromAxis = useCallback(
+        (total: unknown, colIndex: number): ResultValue | null => {
+            const value = last(data.rowTotalFields)?.[colIndex];
+
+            if (!value || !value.fieldId) throw new Error('Invalid pivot data');
+            const item = getField(value.fieldId);
```

Comment:
I think the return type of `ResultValue | null` doesn't match what's happening in the code. What do you think? Should it return something before the `last(...)` call like it's done on `getMetricAsRowColumnTotalValueFromAxis` ? 🤔

Reply:
Changing the new functions type and code to not return null.